portability-transfer/src/main/java/org/dataportabilityproject/transfer/RetryException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.transfer;
+
+public class RetryException extends Exception {
+  private final int attempts;
```

Comment:
can this be documented? i.e. I'm wondering does attempts represent the # we've already made, the number we're allowed to make, etc?

Reply:
Should be clearer now.